.github/workflows/build.yml

```diff
@@ -207,3 +207,143 @@ jobs:
           env_vars: OS,PYTHON,CPLUS
           name: codecov-umbrella
           fail_ci_if_error: false
+
+  build_cuda102-windows:
```

Comment:
the newly added ci failed.

Reply:
will be fixed by https://github.com/open-mmlab/mmdeploy/pull/1626